datastore/src/main/java/io/spine/server/storage/datastore/DsColumnMapping.java

```diff
@@ -120,16 +166,24 @@ protected ColumnTypeMapping<Message, StringValue> ofMessage() {
         return o -> NullValue.of();
     }
 
-    @SuppressWarnings({"ProtoTimestampGetSecondsGetNano", "UnnecessaryLambda"})
-    // This behavior is intended.
-    private static ColumnTypeMapping<Timestamp, TimestampValue> ofTimestamp() {
+    /**
+     * Returns the default mapping from {@link Timestamp} to {@link TimestampValue}.
+     */
+    @SuppressWarnings({"ProtoTimestampGetSecondsGetNano" /* In order to create exact value. */,
```

Comment:
I would move `"ProtoTimestampGetSecondsGetNano"` to a new line. 

And the same with the suppression below.

Reply:
Done.